ParticleObjects/particleDungeon_class.f90

```diff
@@ -423,19 +428,175 @@ subroutine printToFile(self, name)
     integer(shortInt)                  :: i
 
     filename = trim(name)//'.txt'
-    open(unit = 10, file = filename, status = 'new')
+    open(unit = 10, file = filename)
```

Comment:
Why this change?

Reply:
Previously it threw out an error if the file already existed, and if you'd compiled scone to print to a certain file then I found it a bit of a pain to have to delete that file between each simulation, especially if whatever file you're printing to is only of minor importance. But I can add this back in if you think it's important to have.